healthcheck.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Automattic\VIP;
+
+/**
+ * Utility Healthcheck class that runs further than cache-healthcheck,
+ * and allows to verify the app code is working as expected.
+ */
+class Healthcheck {
+	/**
+	 * By default, an app is considered healthy unless filter value returns a non-empty array
+	 *
+	 * @var array
```

Comment:
```suggestion
	 * @var array<string>
```

(Could also be `string[]` like elsewhere in this file, but I prefer the former, as the former feels more like `array()`, and there's nothing like `string[]` syntax in PHP where the type comes first. The former also allows consistency when doing something like `array<int, string>` and other array shapes.)